configs/config-py37.yml

```diff
@@ -1,6 +1,6 @@
 docker_image: "nsls2/linux-anvil-cos7-x86_64:latest"
-env_name: "lsdc-server-2022-2.0-py37"
-conda_env_file: "env-lsdc-server-2022-2.0-py37.yml"
+env_name: "lsdc-server-2022-2.1-py37"
+conda_env_file: "env-lsdc-server-2022-2.1-py37.yml"
```

Comment:
I suggest naming this file as `env-py37.yml`. That way you don't have to update this name each time a new version is created.

The same applies to other env files.

Reply:
good idea, just having the build number in the env_name should be good enough.